src/blog/2024/01/soc2.md

```diff
@@ -0,0 +1,57 @@
+---
+title: FlowFuse is Now SOC 2 Compliant
+subtitle: FlowFuse's Path to SOC 2 Compliance - A Testament to Our Commitment to Security
```

Comment:
It's not security, it's compliance and taking care of customer data. Should it be a commitment to customer and user data?